src/lock_file/resolve/build_dispatch.rs

```diff
@@ -269,12 +269,10 @@ impl<'a> LazyBuildDispatch<'a> {
                 self.prefix_updater.name().as_str()
             );
 
-            let repodata_records = self.repodata_records.as_ref().map_err(|err| {
-                LazyBuildDispatchError::InitializationError(format!(
-                    "failed to get repodata of the building conda prefix: {}",
-                    err
-                ))
-            })?;
+            let repodata_records = self
+                .repodata_records
+                .as_ref()
+                .map_err(|err| LazyBuildDispatchError::InitializationError(format!("{}", err)))?;
```

Comment:
can we do it maybe #[fromerr] ?

Reply:
Ok did it with some help with Bas, unfortunately the error does not show up, as it loses its context coming back out of uv